arrows/mvg/camera_options.h

```diff
@@ -45,6 +45,7 @@ StringToLensDistortionType( std::string value, LensDistortionType& type );
 
 /// Default implementation of string options for cam enums
 template < typename T >
+KWIVER_ALGO_MVG_EXPORT
```

Comment:
```suggestion
```

This line seems to be causing problems with the windows builds.  Let's take it out.